controllers/authrealm/suite_test.go

```diff
@@ -558,12 +561,12 @@ var _ = Describe("Process AuthRealm LDAP: ", func() {
 							IdentityProviderConfig: openshiftconfigv1.IdentityProviderConfig{
 								Type: openshiftconfigv1.IdentityProviderTypeLDAP,
 								LDAP: &openshiftconfigv1.LDAPIdentityProvider{
-									URL:    "myldap.example.com",
+									URL:    "ldap://myldap.example.com:389/dc=example,dc=com?mail,cn?one?(objectClass=person)",
```

Comment:
Shall we build this URL from Filter, BaseDN etc to be more clear how these all tie together? Same comment for ldaps test.